src/views/Login.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import { useRecoilValue } from "recoil";
-import { loggedInState } from "../domain/store";
+import { loggedInState } from "../store";
```

Comment:
fine for now but in future we should prob setup macros instead of relative imports, i.e. @store

Reply:
YES.